link.go

```diff
@@ -28,6 +28,14 @@ type LinkAttrs struct {
 	Namespace    interface{} // nil | NsPid | NsFd
 }
 
+// NewLinkAttrs returns LinkAttrs structure filled with default values
+func NewLinkAttrs() LinkAttrs {
+	return LinkAttrs{
+		TxQLen: -1,
+		MTU:    -1,
+	}
+}
+
```

Comment:
But from that kernel code, it looks like 0 is ok and should be set. `tb[IFLA_TXQLEN` is just `rtattr *`